playbooks/roles/install-butterfly/files/oc-web-terminal.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+hostname=https://{{ openshift_public_hostname }}
+password={{ generic_pass }}
+username={{ generic_user }}
+begin={{ generic_count_begin }}
+count={{ generic_count }}
+pause=5
+
+# Configure openshift.
+# if ! oc whoami &>/dev/null; then
+# printf "Type the server URL that you want to log in to, followed by [ENTER]: "
+#
+# read url
+#
+# if [[ -z "$url" ]]; then
+# printf "OpenShift server URL cannot be null. Please try with a valid URL..."
+# exit 1
+# fi
+#
+# oc login "$url"
+#
+# printf "Type cluster administrator username, followed by [ENTER]: "
+#
+# read admin_user
+#
+# printf "Type cluster administrator password, followed by [ENTER]: "
+#
+# read admin_pass
+# fi
+
+for (( i = $begin; i <= $count; i++ )); do
+ oc login "$hostname" --insecure-skip-tls-verify -u $username${i} -p $password
+
+ # Gather username for project suffix
+ PRJ_SUFFIX=${ARG_PROJECT_SUFFIX:-`echo $(oc $ARG_OC_OPS whoami) | sed -e 's/[-@].*//g'`}
+
+ # # Delete user project.
+ # oc delete project ocwt-$PRJ_SUFFIX
+
+ # Create user project.
+ oc new-project ocwt-$PRJ_SUFFIX
+
+ # Deploy web terminal container.
+ oc new-app quay.io/openshiftlabs/workshop-terminal:2.4.0 -n ocwt-$PRJ_SUFFIX
```

Comment:
checked this. the official supported internal image is openshifthomeroom/workshop-terminal

this should be more like
oc new-app --docker-image=quay.io/openshifthomeroom/workshop-terminal --env=OC_VERSION=3.11 -n ocwt-$PRJ_SUFFIX

I think by default if you don't pass in the oc_version, the client defaults to 4.x

Reply:
Testing now. See https://github.com/epe105/devsecops-workshop/pull/18/commits/394e08de0d584ece7f02709e579b53ccd6189a55